tools/docker/Dockerfile

```diff
@@ -1,6 +1,23 @@
 FROM alpine
 ARG TARGETARCH
-RUN apk add --no-cache --verbose ca-certificates && adduser -D kopia && addgroup kopia kopia
+
+# install CA certificates and create user/group 'kopia' with IDs 51515:51515
+RUN apk add --no-cache --verbose ca-certificates && \
+  addgroup -g 51515 kopia && \
+  adduser -u 51515 -H -D -G kopia kopia && \
+  mkdir -p /logs /cache /config && \
```

Comment:
Kopia should be doing the directory creations in place of it needs them.

Reply:
unless those are under `/home/kopia` (which is ugly and I don't want to create it) it won't have permissions.

Reply:
got rid of all `RUN`